app/models/user.rb

```diff
@@ -24,18 +24,23 @@ def pending_friends
   end
```

Comment:
- [ ] `pending_friends` can be defined as association with SQL query instead of processing objects in application memory (as the database was created to store and query data it is always more efficient than ruby)
Follow the same format as the `confirmed_friendships`